SofaKernel/modules/Sofa.Type/compat/sofa/defaulttype/Vec.h

```diff
@@ -24,7 +24,7 @@
 
 #include <sofa/type/Vec.h>
 
-//SOFA_DEPRECATED_HEADER(v21.12, "sofa/type/Vec.h")
+SOFA_DEPRECATED_HEADER(v21.12, "sofa/type/Vec.h")
```

Comment:
rooh le dawah... 
So from 21.06 to 21.12 we keep the compatibility with a deprecated warning right?
I'll say that if it compiles... fine for me.

Reply:
It is supposed to compile yes 😏
Actually the move has been merged already so if you are compiling fine now, it should be OK still after this merge. 
The only problem which can arise is if you did not include Vec.h before (because impliclty it was included by someone else).
But of course you dont do that in your plugins because **you include what you use**, right ? 😝